images/nginx/rootfs/build.sh

```diff
@@ -20,42 +20,42 @@ set -o pipefail
 
 export NGINX_VERSION=1.20.1
 export NDK_VERSION=0.3.1
-export SETMISC_VERSION=0.32
-export MORE_HEADERS_VERSION=0.33
-export NGINX_DIGEST_AUTH=cd8641886c873cf543255aeda20d23e4cd603d05
-export NGINX_SUBSTITUTIONS=bc58cb11844bc42735bbaef7085ea86ace46d05b
-export NGINX_OPENTRACING_VERSION=0.11.0
-export OPENTRACING_CPP_VERSION=1.6.0
+export SETMISC_VERSION=fddc347134e97d22c34dc5189e76b0ec97bf84d7
+export MORE_HEADERS_VERSION=f85af9649b858e21b400a2150a4c7b8ebd36e921
+export NGINX_DIGEST_AUTH=1.0.0
+export NGINX_SUBSTITUTIONS=b8a71eacc7f986ba091282ab8b1bbbc6ae1807e0
+export NGINX_OPENTRACING_VERSION=2d9f40fa86380b019698f131a803f415ad24c033
+export OPENTRACING_CPP_VERSION=f86b33f3d9e7322b1298ba62d5ffa7a9519c4c41
 export ZIPKIN_CPP_VERSION=f69593138ff84ca2f6bc115992e18ca3d35f344a
-export YAML_CPP_VERSION=yaml-cpp-0.6.3
-export JAEGER_VERSION=0.7.0
-export MSGPACK_VERSION=3.2.1
-export DATADOG_CPP_VERSION=7b560e5c13324c0581476dad3bd8ac4ac5f64045
-export MODSECURITY_VERSION=22e53aba4e3ae8c7d59a3672d6727e49246afe96
-export MODSECURITY_LIB_VERSION=v3.0.4
-export OWASP_MODSECURITY_CRS_VERSION=v3.3.0
-export LUA_NGX_VERSION=138c1b96423aa26defe00fe64dd5760ef17e5ad8
-export LUA_STREAM_NGX_VERSION=0.0.9
-export LUA_UPSTREAM_VERSION=0.07
-export LUA_CJSON_VERSION=2.1.0.8
+export YAML_CPP_VERSION=b591d8ae2ad1ff373273c3e05973adf6c46abfa8
+export JAEGER_VERSION=e90642a250289623fc13f650991d108e9a19b6b1
+export MSGPACK_VERSION=3.3.0
+export DATADOG_CPP_VERSION=5e3e3571d2341883a15fc4b5b443f930cc70e8c1
+export MODSECURITY_VERSION=2497e6ac654d0b117b9534aa735b757c6b11c84f
+export MODSECURITY_LIB_VERSION=v3.0.5
+export OWASP_MODSECURITY_CRS_VERSION=v3.3.2
+export LUA_NGX_VERSION=66a4d417d654404ade32fcc600ff1381e5e02f9c
+export LUA_STREAM_NGX_VERSION=b977cee61a1cc0ea988a581cf0adbc955c9a95ab
+export LUA_UPSTREAM_VERSION=8aa93ead98ba2060d4efd594ae33a35d153589bf
+export LUA_CJSON_VERSION=4b350c531de3d71008c77ae94e59275b8371b4dc
 export NGINX_INFLUXDB_VERSION=5b09391cb7b9a889687c0aa67964c06a2d933e8b
-export GEOIP2_VERSION=3.3
-export NGINX_AJP_VERSION=bf6cd93f2098b59260de8d494f0f4b1f11a84627
-
-export LUAJIT_VERSION=2.1-20201027
-
-export LUA_RESTY_BALANCER=af4508f7aa5560c7d810922c2515b557f9e5d51a
-export LUA_RESTY_CACHE=0.10
-export LUA_RESTY_CORE=0.1.21
-export LUA_RESTY_COOKIE_VERSION=766ad8c15e498850ac77f5e0265f1d3f30dc4027
-export LUA_RESTY_DNS=0.21
-export LUA_RESTY_HTTP=0.15
-export LUA_RESTY_LOCK=0.08
-export LUA_RESTY_UPLOAD_VERSION=0.10
-export LUA_RESTY_STRING_VERSION=0.12
-export LUA_RESTY_MEMCACHED_VERSION=0.15
-export LUA_RESTY_REDIS_VERSION=0.29
-export LUA_RESTY_IPMATCHER_VERSION=1a0a1c58fd085b15eedee58de8b5f45c27aff7bc
+export GEOIP2_VERSION=a26c6beed77e81553686852dceb6c7fdacc5970d
+export NGINX_AJP_VERSION=a964a0bcc6a9f2bfb82a13752d7794a36319ffac
+
+export LUAJIT_VERSION=2.1-20210510
```

Comment:
Whats with this one @rikatz? https://github.com/openresty/luajit2/compare/v2.1-20210510...v2.1-agentzh shows that has been a lot going on since the last release.

Reply:
Does not seem to be an official release :(

https://github.com/openresty/luajit2/releases

This is why I've pointed to the release

Reply:
Sure, I kept using the 2.1-20210510. But v2.1-agentzh seems to be the main branch. So after release 2.1-20210510 there was some commits going on. Or am I wrong?!.

Reply:
you are right :) I just wanted to stick with the releases specifically for luajit2 due to its critical role :D